lib/migration-stream.js

```diff
@@ -31,7 +32,13 @@ function MigrationStream() {
     if (v2 === true) {
       // Check for errors on the v2 tile
       var err = checkForErrors(info);
-      if (err) return callback(err);
+      if (err) {
+        if (!invalidTileLogged && process.env.LOG_INVALID_VT){
+          fs.writeFileSync('vt-invalid.json', JSON.stringify({ 'err': err }));
+          invalidTileLogged = true;
+        }
+        return callback(err)
```

Comment:
I'm curious, is the `invalidTileLogged` boolean needed to avoid this getting called twice in some scenario? Would it get called twice in the case that an error is thrown for one tile but the program does not exit immediately before another tile is processed?

Reply:
@springmeyer you're right! We don't `invalidTileLogged` - it should return the error before another tile processes. Just removed it!